docker-compose.yml

```diff
@@ -1,42 +1,47 @@
-nginx:
-    image: nginx
-    ports:
-        - "8080:80"
-    volumes:
-        - ./docker/nginx/nginx.conf:/nginx.conf
-        - ./docker/logs/nginx:/var/log/nginx
-        - .:/var/www/html
-    links:
-        - php:php
-    command: nginx -c /nginx.conf
-php:
-    build: docker/php
-    ports:
-        - "9000:9000"
-    volumes:
-        - .:/var/www/html
-    #links:
-        # - "postgres:rdbms"
-        # - "mariadb:rdbms"
-    env_file:
-        - ./docker/php/env
-        # Comment non-used DBMS lines
-        # If all DBMS are commented out, sqlite will be used as default
-        # - ./docker/postgres/env
-        # - ./docker/mariadb/env
-#postgres:
-#    image: postgres:9
-#    ports:
-#        - "5432:5432"
-#    volumes:
-#        - ./docker/data/pgsql:/var/lib/postgresql/data
-#    env_file:
-#        - ./docker/postgres/env
-#mariadb:
-#    image: mariadb:10
-#    ports:
-#        - "3306:3306"
-#    volumes:
-#        - ./docker/data/mariadb:/var/lib/mysql
-#    env_file:
-#        - ./docker/mariadb/env
+version: '2'
+services:
+    nginx:
+        image: nginx
+        ports:
+            - "8080:80"
+        volumes:
+            - ./docker/nginx/nginx.conf:/nginx.conf
+            - ./docker/logs/nginx:/var/log/nginx
+            - .:/var/www/html
+        links:
+            - php:php
+        command: nginx -c /nginx.conf
+    php:
+        build:
+            context: docker/php
+            args:
+                timezone: 'Europe/Monaco'
```

Comment:
I did it to show that the argument in the compose file is the one used.
The one in the dockerfile is a default one.
So like this, and as someone rightly commented, only the compose file need
to be modified.

I hope it's clear like that.
Le 3 mars 2016 08:56, "Jeremy Benoist" <notifications@github.com> a écrit :

> In docker-compose.yml
> <https://github.com/wallabag/wallabag/pull/1749#discussion_r54846183>:
>
> > +    nginx:
> > +        image: nginx
> > +        ports:
> > +            - "8080:80"
> > +        volumes:
> > +            - ./docker/nginx/nginx.conf:/nginx.conf
> > +            - ./docker/logs/nginx:/var/log/nginx
> > +            - .:/var/www/html
> > +        links:
> > +            - php:php
> > +        command: nginx -c /nginx.conf
> > +    php:
> > +        build:
> > +            context: docker/php
> > +            args:
> > +                timezone: 'Europe/Monaco'
>
> Why did you put Monaco here and Paris in Dockerfile?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wallabag/wallabag/pull/1749/files#r54846183>.
>